docs/en/02_Creating_pages_and_content/01_Creating_And_Editing_Content/04_Images_And_Documents.md

```diff
@@ -70,6 +70,12 @@ Basic functionality and options for files:
 
 ![Inserted image](../../_images/inserted-image.png)
 
+## File usage
+
+The Used on tab allows a CMS content author to view the usage of a file across the CMS. Click the Used on tab in the File details panel. The first row shows this image being used on a content block and the page this content block is used on.  Click links in each row to navigate to the relevant edit screen in the CMS.
```

Comment:
```suggestion
The Used on tab allows a CMS content author to view the usage of a file across the CMS. Click the ***Used on*** tab in the File details panel. The first row shows this image being used on a content block and the page this content block is used on.  Click links in each row to navigate to the relevant edit screen in the CMS.
```

Reply:
You need to work on this sentence. 
- Link the first mention of 'Content block' to the elemental User help page.

Something along the lines of:
> The first row shows this file is being used on a [Content block](link) and also provides the Page name of where the Content block has been added in the CMS. To navigate to where a particular file is being used, click on the relevant row.

We shouldn't add the screenshot here until the feature has been completed and been through a peer design channel. As this isn't matching currently e.g font-size.

Reply:
Cheers, have updated text as recommended

Have added a TODO in the description of this PR to update the screenshot after the linked PR has been approved and merged